boot/initramfs.go

```diff
@@ -83,3 +86,26 @@ func EnsureNextBootToRunMode(systemLabel string) error {
 	}
 	return bl.SetBootVars(m)
 }
+
+// initramfsReboot triggers a reboot from the initramfs immediately
+var initramfsReboot = func() error {
+	if osutil.IsTestBinary() {
+		panic("initramfsReboot must be mocked in tests")
+	}
+
+	out, err := exec.Command("/bin/reboot").CombinedOutput()
+	if err != nil {
+		return osutil.OutputErr(out, err)
+	}
+	// should be unreachable
+	return nil
```

Comment:
anyway I think if we think this should be unreachable we should have a panic here

Reply:
done